streams/src/main/java/org/apache/kafka/streams/kstream/PrintForeachAction.java

```diff
@@ -34,13 +34,30 @@
      * @param streamName The given name will be printed.
      */
     public PrintForeachAction(final PrintWriter printWriter, final String streamName) {
+        this(printWriter, null, streamName);
```

Comment:
Only `KTableImpl.print` and `KTableImpl.writeAsText` use this `PrintForeachAction(final PrintWriter printWriter, final String streamName)`, so I decide to delete this one.
It will make code more clean :)